public tests now fail with following error, but i am sure that it is not related to original problem
 
```
--- FAIL: TestCallContextResult (2.43s)
	Error Trace:	rpc_test.go:186
	Error:      	Should be true
	Messages:   	balance should be higher than 0
--- FAIL: TestRPCTestSuite (11.07s)
```

--- a/e2e/rpc/rpc_test.go
+++ b/e2e/rpc/rpc_test.go
@@ -181,7 +181,7 @@ func (s *RPCTestSuite) TestCallContextResult() {
 	defer cancel()
 
 	var balance hexutil.Big
-	err := client.CallContext(ctx, &balance, "eth_getBalance", "0xbF164ca341326a03b547c05B343b2E21eFAe24b9", "latest")
+	err := client.CallContext(ctx, &balance, "eth_getBalance", TestConfig.Account1.Address, "latest")
 	s.NoError(err)
 	s.True(balance.ToInt().Cmp(big.NewInt(0)) > 0, "balance should be higher than 0")
 }

--- a/geth/node/manager.go
+++ b/geth/node/manager.go
@@ -299,6 +299,9 @@ func (m *NodeManager) addPeer(url string) error {
 }
 
 func (m *NodeManager) PeerCount() int {
+	if m.node == nil || m.node.Server() == nil {
+		return 0
+	}
 	return m.node.Server().PeerCount()
 }
 

